sentry_sdk/tracing.py

```diff
@@ -1583,6 +1583,22 @@ def set_attribute(self, key, value):
 
         self._otel_span.set_attribute(key, _serialize_span_attribute(value))
 
+    @property
+    def status(self):
+        # type: () -> Optional[str]
+        if not hasattr(self._otel_span, "status"):
+            return None
+
+        if self._otel_span.status.status_code == StatusCode.OK:
+            return SPANSTATUS.OK
+        else:
+            return SPANSTATUS.UNKNOWN_ERROR
```

Comment:
Are we taking into account span status potentially being unset here? Is this covered by the first check of the span not having any `status` attr or is there a specific `StatusCode` for it?